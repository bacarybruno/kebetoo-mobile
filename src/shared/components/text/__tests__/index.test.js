import setupTest from 'Kebetoo/src/config/jest-setup'

import Text from '../index'

const givenText = setupTest(Text)({
  text: 'Hello World',
})

it('renders Text', () => {
  const { wrapper } = givenText()
  expect(wrapper.toJSON()).toMatchSnapshot()
})
