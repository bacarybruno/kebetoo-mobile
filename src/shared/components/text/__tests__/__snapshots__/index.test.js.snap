// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Text 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Helvetica",
      },
      Object {
        "color": "rgba(0, 0, 0, 0.8)",
        "fontSize": 16,
        "fontWeight": "normal",
        "opacity": 1,
        "textDecorationLine": "none",
      },
      Object {},
    ]
  }
>
  Hello World
</Text>
`;
